analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -597,3 +586,12 @@ open class Npm(
 
 /** Name of the configuration option to toggle legacy peer dependency support. */
 private const val OPTION_LEGACY_PEER_DEPS = "legacyPeerDeps"
+
```

Comment:
Commit message: "compan~~t~~ion"
I am not aware of that convention, do we have it documented somewhere?

Reply:
> I am not aware of that convention, do we have it documented somewhere?

Not in written form. It was a proposal that @fviernau came up with and that we verbally agreed on, AFAIR.

Reply:
I think we really need to start documenting those, let's talk about it in the next TSC meeting.

Reply:
> I think we really need to start documenting those

Would `CONTRIBUTING.md` be a good place for that?

Reply:
Yes. Except if we want different rules for different repositories, but it would be a start.